clippy.toml

```diff
@@ -0,0 +1,56 @@
+msrv = "1.67"
+
+
+# https://rust-lang.github.io/rust-clippy/master/index.html#disallowed_macros
+disallowed-macros = [
+  'dbg',
+
+  # TODO(emilk): consider forbidding these to encourage the use of proper log stream, and then explicitly allow legitimate uses
+  # 'std::eprint',
+  # 'std::eprintln',
+  # 'std::print',
+  # 'std::println',
+
+  # 'std::unimplemented', # generated by ArrowDeserialize derive-macro :(
+]
+
+# https://rust-lang.github.io/rust-clippy/master/index.html#disallowed_methods
+disallowed-methods = [
+  "std::env::temp_dir", # Use the tempdir crate instead
+
+  # Disabled because of https://github.com/rust-lang/rust-clippy/issues/10406
+  #   "std::time::Instant::now",      # use `instant` crate instead for wasm/web compatability
```

Comment:
I hit this before and would be thankful for the clippy warning. Why can't we enable this warning yet?

Reply:
Because `instant::Instant` is an alias of `std::time::Instant` on native: https://github.com/rust-lang/rust-clippy/issues/10406#issuecomment-1445409755